README.md

```diff
@@ -45,3 +49,6 @@ for more details.
 [cpplint]: https://github.com/google/styleguide/tree/gh-pages/cpplint
 [emacs]: https://raw.githubusercontent.com/google/styleguide/gh-pages/google-c-style.el
 [xml]: https://google.github.io/styleguide/xmlstyle.html
+[dart]: https://www.dartlang.org/guides/language/effective-dart
+[go]: https://github.com/golang/go/wiki/Style
```

Comment:
Yeah, it is a bit weirdly named :) You can use the golang.org shortcut for the link btw: https://golang.org/wiki/CodeReviewComments. If we ever move the wiki somewhere, the link would still work.